### PR TITLE
Feat: added home

### PR DESCRIPTION
**Pull request by Rohanmond**
File structure:
pages=> home=>home.html, home.css
Screenshot 2022-02-12 at 12 17 00 PM

### CODE REVIEW

**Review by hrishi82 — APPROVED** [review]
Looks good!

File: doc_style/doc_landing.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  }
+}
+.landing-container {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+}
+.quiz-hero-logo {
+  max-width: 5rem;
+  /* max-height: 5rem; */
```
Comment (anuj-kumary) [inline doc_style/doc_landing.css]: Remove unwanted comments

File: index.html [hunk 2]
```diff
@@ -7,5 +7,41 @@
...
+  <body>
+    <div class="landing-container">
+      <nav class="navigation landing-nav">
+        <div class="nav-left">
+          <div class="nav-logo-container">
+            <a href="#">
+              <img
+                class="hero-logo quiz-hero-logo"
+                src="./src/logo/DevQuiz.png"
+                alt=""
```
Comment (anuj-kumary) [inline index.html:19]: Give some **alt** value don't keep blank

File: pages/home/home.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        <h1>Dev Quiz</h1>
+      </div>
+      <div class="home-buttons">
+        <button class="home-cat-button">Easy</button>
+        <button class="home-cat-button">Medium</button>
+        <button class="home-cat-button">Hard</button>
+      </div>
+      <div class="home-cards">
+        <div class="quiz-card brd-rd-semi-sq">
+          <div class="quiz-card-img-container brd-rd-semi-sq">
```
Comment (Shah-Arsalan) [inline pages/home/home.html]: Naming can be better

**Review by Shah-Arsalan — APPROVED** [review]

File: doc_style/doc_landing.css (hunk 1 above)
Comment (Rohanmond) [reply doc_style/doc_landing.css]: it is fixed now

File: index.html (hunk 2 above)
Comment (Rohanmond) [reply index.html:19]: it is fixed now

File: pages/home/home.html (hunk 3 above)
Comment (Rohanmond) [reply pages/home/home.html]: it is fixed now

**Pull request merged**